File: trunk/SuperShooter/Assets/Scripts/Zombie Spawner.js
var zombie : GameObject;
var spawnRate = 5;

private var lastzombiespawn = 0; 


function Awake (){



}


function Update () {

	var dif ;
	dif = Time.time-lastzombiespawn;

	if (dif > spawnRate){
		var position = Vector3(Random.Range(100, 200), 10, Random.Range(100, 200));
		var newZombie = Instantiate(zombie, position, Quaternion.identity);
		var zombieAI = newZombie.GetComponent("ZombieAI");
		
		lastzombiespawn = Time.time;
	}

}

File: SuperShooter/Assets/Scripts/ZombieSpawner.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class ZombieSpawner : MonoBehaviour
{
    public GameObject zombie;
    public int spawnRate;
    private int lastzombiespawn;
    public virtual void Awake()
    {
    }

    public virtual void Update()
    {
        float dif = Time.time - this.lastzombiespawn;
        if (dif > this.spawnRate)
        {
            Vector3 position = new Vector3(Random.Range(100, 200), 10, Random.Range(100, 200));
            GameObject newZombie = UnityEngine.Object.Instantiate(this.zombie, position, Quaternion.identity);
            Component zombieAI = newZombie.GetComponent("ZombieAI");
            this.lastzombiespawn = (int) Time.time;
        }
    }

    public ZombieSpawner()
    {
        this.spawnRate = 5;
    }

}